torch/csrc/DeviceAccelerator.cpp

```diff
@@ -0,0 +1,82 @@
+#include <c10/core/DeviceGuard.h>
+#include <torch/csrc/DeviceAccelerator.h>
+#include <torch/csrc/utils/device_lazy_init.h>
+
+namespace torch::accelerator {
+
+void initModule(PyObject* module) {
+  auto m = py::handle(module).cast<py::module>();
+
+  m.def("_accelerator_getAccelerator", []() {
+    // If no accelerator is currently available, return CPU.
+    return c10::Device(at::getAccelerator(false).value_or(c10::kCPU));
```

Comment:
From discussion, feels like we shouldn't do or cpu here and fail.
We can easily add this back later if it is feels really needed (going from error -> cpu is not BC-breaking).

Reply:
Sounds good. I also think at this stage self-consistent is more important.